fenicsxprecice/fenicsxprecice.py

```diff
@@ -62,52 +62,28 @@ def __init__(self, adapter_config_filename='precice-adapter-config.json'):
         self._dofmap = None  # initialized later using function space provided by user
 
         # coupling mesh related quantities
-        self._owned_vertices = Vertices(VertexType.OWNED)
-        self._unowned_vertices = Vertices(VertexType.UNOWNED)
         self._fenicsx_vertices = Vertices(VertexType.FENICSX)
```

Comment:
See comment above: I think we don't need any type for the vertices, if there are only vertices of `VertexType.FENICSX`.